### DISCUSSION TITLE
Add “test email send” button to SMTP email setup

### DISCUSSION THREAD

**Discussion by pkellner** [coollabsio/coolify#3282] [Improvement Requests]
it would be nice to be able to test the set up after entering SMTP information. I added mine and had no idea if it would work. I was happy to receive an email when my build was successful, so that’s how I knew.

**Comment by jorgepazp** [comment]
i second this.
by the way could please show an example of a correct smtp configuration? I've been trying to set up mine but I think I may be missing onto something

**Comment by fcpauldiaz** [comment]
this already exists from a long time ago. Notifications > Send Test Email

**Reply by pkellner** [reply]
> this already exists from a long time ago. Notifications > Send Test Email

Screen shot please, I could not find it.

**Reply by fcpauldiaz** [reply]
![Screenshot 2024-09-10 at 10 47 54 AM](https://github.com/user-attachments/assets/aeef2476-fa03-4b37-8425-ff9c10b55a4f)

**Reply by pkellner** [reply]
> > this already exists from a long time ago. Notifications > Send Test Email
> 
> Screen shot please, I could not find it.

I did find it there later while setting up notifications. However, that is not related to my suggestion.  I was specifically suggesting add the "send test email" button to the page where you put in your mail information so you can know if it will work. That would be very helpful (and kind of expected).

![image](https://github.com/user-attachments/assets/5c63d414-6206-43c9-8c44-b361cbfa3d32)
